README.md

```diff
@@ -1,25 +1,16 @@
-# Go Fn Development Kit (FDK)
-
 [![GoDoc](https://godoc.org/github.com/fnproject/fdk-go?status.svg)](https://godoc.org/github.com/fnproject/fdk-go)
 
-fdk-go provides convenience functions for writing Go fn code
-
-For getting started with fn, please refer to https://github.com/fnproject/fn/blob/master/README.md
-
-# Installing fdk-go
-
-```sh
-go get github.com/fnproject/fdk-go
-```
-
-or your favorite vendoring solution :)
+# Go FDK Documentation
+This is documentation for the Go function development kit (FDK). The Go FDK  provides convenience functions for writing Go Fn code.
 
-# Examples
+## User Information
```

Comment:
We do need a link to the top of the doc set for Fn, but one of these links are actually related to the Go FDK.  What we need is something on how to obtain the Go FDK.  There are two options: 1) it's in the build and runtime Go images, and 2) if someone were building a custom runtime image using a Dockerfile, how do they include it in their program dependencies?

Reply:
Adding issue and doc updates. Will provide that info shortly.

Reply:
Addressing the above by adding issues to fdk-go:

* Add https://fnprojec.io to repo description at top of the page. See: https://github.com/fnproject/fdk-go/issues/47
* Add new content on Go FDK development. See: ttps://github.com/fnproject/fdk-go/issues/48
* This PR also closes: https://github.com/fnproject/fdk-go/issues/40